packages/holocron/src/RenderModule.jsx

```diff
@@ -29,6 +29,8 @@ export default function RenderModule({ children, moduleName, props }) {
 
   const propsWithChildren = typeof children !== 'undefined' ? { children, ...props } : props;
 
+  // disabling this because propsWithChildren is computed and passed down to this prop
```

Comment:
maybe
```suggestion
  // propsWithChildren is computed and passed down to this prop
```
for brevity? #nitpick